sdk/tests/js/tests/tex-image-and-sub-image-2d-with-image.js

```diff
@@ -43,7 +43,7 @@ function generateTest(pixelFormat, pixelType, pathToTestRoot, prologue) {
         gl.clearColor(0,0,0,1);
         gl.clearDepth(1);
 
-        wtu.loadTexture(gl, "/resources/red-green.png", runTest);
+        wtu.loadTexture(gl, "../../resources/red-green.png", runTest);
```

Comment:
The above comment is for the two missing files in the commit.

As for the URL change, the wtu.loadTexture simply put the URL as Image.src, so it has to be relative.  I agree it's confusing because now the js code and the html are in different locations.  Maybe we can modify the wtu.loadTexture() code and always load from resources/ folder? So even the "resources" in the path can be avoided.